recipes/_packaging.rb

```diff
@@ -58,3 +62,5 @@
   omnibus_env['PATH'] << node['wix']['home']
   omnibus_env['PATH'] << node['7-zip']['home']
 end
+
+include_recipe 'omnibus::_libffi' if ppc64? || ppc64le?
```

Comment:
Should this include_recipe appear in a different recipe? What do we need LibFFI for? It doesn't seem packaging related.